packages/arb-token-bridge-ui/package.json

```diff
@@ -112,6 +116,6 @@
     "tailwindcss": "^3.2.4",
     "ts-node": "^10.9.1",
     "ts-prune": "^0.10.1",
-    "typescript": "^4.7.4"
+    "typescript": "^5.2.2"
```

Comment:
had to bump it otherwise it won't work with viem
> Types currently require using TypeScript v5.0.4 or greater.
https://viem.sh/docs/typescript.html

Reply:
keeping it as it's good to bump packages